ballerina-tests/build.gradle

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/e99db12aa0dc4da0c43e0d716cd183096359bf21